src/terms/xc.jl

```diff
@@ -94,16 +94,16 @@ libxc provides the scalars
     Vρ = ∂(ρ E)/∂ρ
     Vσ = ∂(ρ E)/∂σ
 
-Consider a variation dϕi of an orbital (considered real for
-simplicity), and let dEtot be the corresponding variation of the
+Consider a variation δϕi of an orbital (considered real for
+simplicity), and let δEtot be the corresponding variation of the
 energy. Then the potential Vxc is defined by
-    dEtot = 2 ∫ Vxc ϕi dϕi
+    δEtot = 2 ∫ Vxc ϕi δϕi
```

Comment:
This d -> δ replacement does not blindly applies in this file. Did you guys check that makes sense everywhere to do this? Like in this case I would stick with `d`. But I'm not the mathematician here ;)

Reply:
I think that here the integration variable `dx` have been muted, I did check every change (not blind search and replace :p) and I think this one makes sense cause we compute `Vxc` against `ϕi` against `δϕi` which results in `δEtot`. But maybe I am mistaking @antoine-levitt

Reply:
I think it does make sense that we use consistently \delta x to mean "infinitesimal variation of x", no? In mathematics people don't use this convention at all but instead give completely different names for the variation, which is dumb for physical objects.

Reply:
Ok. Than I agree. For me the integration was "with respect to the \psi_i", but one could also view it as the dx being muted, I agree with that @gkemlin.  check.